app/views/documents/_new.erb

```diff
@@ -0,0 +1,19 @@
+<style>
+    .ui-autocomplete {
+        z-index: 9999;
+    }
+</style>
+
+<div class="employee-upload">
+  <i class="fa fa-times fa-3x pull-right modal-close"></i>
+
+  <div class="upload-preview">
+
+  </div>
+  <%= form_for @document, :url => url_for(:controller => 'documents', :action => 'create'), method: 'post', :html => { :multipart => true } do |f| %>
```

Comment:
<img width="737" alt="welcome_to_health_connector___health_connector" src="https://user-images.githubusercontent.com/599739/26978004-d4265bf4-4cf7-11e7-824e-b0aeaba88f3b.png">

This is how the modal opens for me